ansible/roles/controller/tasks/deploy.yml

```diff
@@ -363,6 +365,53 @@
   include_tasks: "lean.yml"
   when: lean
 
+# Before redeploy controller, should remove that controller instance from nginx
+- name: remove the controller from nginx's upstream configuration
+  shell:
+    docker exec -t nginx sh -c "sed -i  \"s/ server {{ ansible_host }}:{{ controller.basePort + (controller_index | int) }}/ \#server {{ ansible_host }}:{{ controller.basePort + (controller_index | int) }}/g\"  /etc/nginx/nginx.conf  && nginx -s reload"
```

Comment:
Personally, I don't like this approach, but this is a kind of limitation because opensource nginx does not support active health check by default.
It is related to https://github.com/apache/openwhisk/issues/5337